tests/fips/gnutls/gnutls_server.pm

```diff
@@ -0,0 +1,93 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: SLES15SP2 FIPS certification, we need to certify gnutls and libnettle
+#          In this case, will configure GnuTLS server
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#63223, tc#1744099
+
+use base "consoletest";
+use testapi;
+use strict;
+use warnings;
+
+sub run {
+    select_console "root-console";
+
+    # Create test folder
+    my $test_dir = "gnutls";
+    assert_script_run "mkdir $test_dir && cd $test_dir";
+
+    # Add support for X.509.  First we generate a CA
+    my $pri_ca_key   = "x509-ca-key.pem";
+    my $ca_tmpl      = "ca.tmpl";
+    my $ca_tmpl_cont = <<'EOF';
+cn = GnuTLS test CA
```

Comment:
I have no idea why the source code color is not showing correctly, But the CI check is passed.
http://10.67.17.9/tests/153/modules/gnutls_server/steps/1/src

Reply:
Yes, I hit the same issue, but it is not code issue, it might caused by browser. since no issue with CI check, and no issue on my editor as well.

Reply:
okay, as we discussed, it is a bit weird that CI check did not get the error, but it looks no harm for the running test.